packages/contracts-bedrock/scripts/checks/hotfuzz/main.go

```diff
@@ -0,0 +1,161 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os/exec"
+	"regexp"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+)
+
+type TestInfo struct {
+	Name     string
+	Contract string
+}
+
+type TestConfig struct {
+	maxSeconds  int
+	maxRuns     int
+	matchPaths  string
+	currentRuns int
+}
+
+func main() {
```

Comment:
I'm running `go run ./scripts/checks/hotfuzz/main.go --max-runs 100 --match-path ./test/L1/OptimismPortal2.t.sol` and it's hanging forever so might need to debug that

Reply:
Checking it out, does testing with `--match-path` testing with `--max-runs` and `--max-seconds` work for you? I can confirm that one works on my end, it takes a lot of time since its ~4 minutes to gather all the test names